oauth/src/lib.rs

```diff
@@ -162,8 +212,212 @@ fn get_socket_address(redirect_uri: &str) -> Option<SocketAddr> {
     None
 }
 
+/// Struct that handle obtaining and refreshing access tokens.
+///
+/// Should not be instantiate by itself, use [`OAuthClientBuilder`] instead.
+pub struct OAuthClient {
+    scopes: Vec<String>,
+    redirect_uri: String,
```

Comment:
```suggestion
    redirect_uri: String, // must match what is registered to the client ID
```
I don't know the correct way to make the docs pick this comment up

Reply:
If it's not public (which this isn't) it is not required and will not be shown anyways. You would want to add the documentation where you initialize the value, which would be the `Builder`^^

Reply:
Sounds like a better place, thanks.